packages/ui/hooks/usePoolInfo.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+
+export type usePoolInfoData = {
+  markets: {
+    [marketAddress: string]: {
+      name?: string;
+    };
+  };
+};
+export function usePoolInfo(chainId?: string, comptrollerAddress?: string) {
```

Comment:
Maybe we should start calling these hooks `useAPIComptroller`

We have a lot of hooks and it's confusing which does what. Quite sure we have some hooks which are called PoolData or so as well.

So let's start using names likes `useAPI` for our api related hooks.

Reply:
Fixed